### PR TITLE
Colorize `any` in class/interface.

### PR DESCRIPTION
**Pull request by hoanhtien**
Fixes https://github.com/Microsoft/TypeScript-Sublime-Plugin/issues/358.

### CODE REVIEW

File: TypeScript.YAML-tmLanguage [hunk 1]
```diff
@@ -461,7 +461,7 @@ repository:
 
   storage-keyword:
     name: storage.type.ts
-    match: \b(number|boolean|string|var|let|function|const)\b
+    match: \b(number|boolean|string|any|var|let|function|const)\b
```
Comment (DanielRosenwasser) [inline TypeScript.YAML-tmLanguage]: Does the JSX file need to be updated as well?

Comment (hoanhtien) [inline TypeScript.YAML-tmLanguage]: Oh right, JSX does need to be updated.

Comment (hoanhtien) [inline TypeScript.YAML-tmLanguage]: Updated.

**Comment by zhengbli** [comment]
:+1:

**Pull request merged**